bindings/python/tests/test_parse.py

```diff
@@ -32,3 +32,60 @@ def get_parsed_info(tmp_path, file_contents):
     )
 
     return parsed_info
+
+
+def test_class_template(tmp_path):
+    file_contents = """
+    template <typename T>
+    struct AStruct {};
+    """
+    parsed_info = get_parsed_info(tmp_path=tmp_path, file_contents=file_contents)
+
+    class_template = parsed_info["members"][0]
+
+    assert class_template["kind"] == "CLASS_TEMPLATE"
+    assert class_template["name"] == "AStruct"
```

Comment:
Should we test the name of the template parameter link in non-type parameter test?

Reply:
Added